include/regs.h

```diff
@@ -42,7 +57,7 @@
 #define kREG(r) BASE_REG(27, r)
 #define bREG(r) BASE_REG(28, r)
 
-/* TODO: Actually confirm these, in case of miss-match it's at least a simple list to `sed` */
+/* TODO: Actually confirm these */
 #define R_TIME_SPEED                      REG(15)
 #define R_RUN_SPEED_LIMIT                 REG(45)
 
```

Comment:
```suggestion
/* TODO: Actually confirm these */

#define R_TIME_SPEED                      REG(15)
#define R_RUN_SPEED_LIMIT                 REG(45)

```
Could you put a space here? I know this comment is referring to all the regs and not just these two since some of the earlier ones were copied from OoT and may not be applicable to MM. Might even be worth clarifying this comment since many of the regs were properly named (but not all)